care/facility/models/bed.py

```diff
@@ -39,3 +40,9 @@ class AssetBed(BaseModel):
     asset = models.ForeignKey(Asset, on_delete=models.PROTECT, null=False, blank=False)
     bed = models.ForeignKey(Bed, on_delete=models.PROTECT, null=False, blank=False)
     meta = JSONField(default=dict)
+
+
+class ConsultationBed(BaseModel):
+    consultation = models.ForeignKey(PatientConsultation, on_delete=models.PROTECT, null=False, blank=False)
+    bed = models.ForeignKey(Bed, on_delete=models.PROTECT, null=False, blank=False)
```

Comment:
We need a start date and end date for each consultation bed assignment, it should also be helpful if we can have a current bed association from the consultation for ease of queries. 
This start and end date would be used to identify when the patient has entered/ exited specific bed types